data-release-pipeline/prepare_for_molgenis/ArgumentParser.py

```diff
@@ -0,0 +1,45 @@
+import argparse
```

Comment:
Why not make this part of the data release bash script instead of introducing new scripts in a different language?

Reply:
Because there are already several scripts written in python and writing bash takes me days, rather than hours. I simply didn't have the time to do that and I think a script is better than a written out explanation about how to do it by hand